ambry-network/src/main/java/com.github.ambry.network/Transmission.java

```diff
@@ -132,8 +132,7 @@ public SocketAddress getRemoteSocketAddress() {
   /**
    * Close the connection for the socket channel
    */
-  public abstract void close()
-      throws IOException;
```

Comment:
@vgkholla : So, how can we justify that the abstract method needs to throw IOException, when none of the inherited methods dont.